lighthouse-core/audits/installable-manifest.js

```diff
@@ -51,59 +125,73 @@ class InstallableManifest extends MultiCheckAudit {
   }
 
   /**
-   * @param {LH.Artifacts.ManifestValues} manifestValues
-   * @return {Array<string>}
+   * @param {LH.Artifacts} artifacts
+   * @return {Array<LH.IcuMessage>}
    */
-  static assessManifest(manifestValues) {
-    if (manifestValues.isParseFailure && manifestValues.parseFailureReason) {
-      return [manifestValues.parseFailureReason];
+  static getInstallabilityErrors(artifacts) {
+    const installabilityErrors = artifacts.InstallabilityErrors.errors;
+
+    const errorMessages = [];
+    for (const err of installabilityErrors) {
+      // @ts-expect-error errorIds from protocol should match up against the strings dict
```

Comment:
should definitely check that the `errorIds`s are in there because there's no reason to assume that this will always be true